src/get-started/timestamping-tutorial.md

```diff
@@ -903,40 +899,40 @@ performed in the directory containing `Cargo.toml`.
    ```shell
    exonum-demo-timestamping finalize \
        --public-api-address 0.0.0.0:8200 \
-       --private-api-address 0.0.0.0:8091 example/sec_1.toml \
-       example/node_1_cfg.toml \
+       --private-api-address 0.0.0.0:8091 example/1/sec_1.toml \
+       example/1/node_1_cfg.toml \
        --public-configs \
-       example/pub_1.toml \
-       example/pub_2.toml \
-       example/pub_3.toml \
-       example/pub_4.toml
+       example/1/pub_1.toml \
+       example/1/pub_2.toml \
+       example/1/pub_3.toml \
+       example/1/pub_4.toml
```

Comment:
It is incorrect and can be done in more simple way, see [PR 1292](https://github.com/exonum/exonum/pull/1292/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR35)